### PR TITLE
Indent ternaries

### PR DESCRIPTION
**Pull request by rattrayalex**
Fixes the part of https://github.com/jlongster/prettier/issues/317 not covered by https://github.com/jlongster/prettier/pull/473 as mentioned in  https://github.com/jlongster/prettier/issues/467#issuecomment-275476597.

Should probably wait for #473 to merge first, as this will have conflicts in the test output. Alternatively, would be happy to just have @vjeux add this commit to his PR.

### CODE REVIEW

**Comment by jlongster** [comment]
Nice! I noticed this in a few other cases too, this makes sense. Checking out #473 right now.

**Comment by jlongster** [comment]
Argh, snapshots conflicts. Can you rebase?

**Comment by vjeux** [comment]
👍

**Comment by rattrayalex** [comment]
Sweet, rebased and re-ran `jest -u`.

File: tests/jsx/__snapshots__/jsfmt.spec.js.snap [hunk 1]
```diff
@@ -156,15 +156,15 @@ exports[`test hug.js 1`] = `
...
-    </div>}
+        {routes.map(route => (
+          <MatchAsync
+            key={\`\${route.to}-async\`}
+            pattern={route.to}
+            exactly={route.to === \"/\"}
+            getComponent={routeES6Modules[route.value]}
+          />
+        ))}
+      </div>}
```
Comment (rattrayalex) [inline tests/jsx/__snapshots__/jsfmt.spec.js.snap]: I do want to call this out – currently, this outputs the closing `}` on the final line, which is indented. That might make it a bit harder to scan a JSX doc and see where a conditional ends. 

On the other hand, I've always found the dangling `}` annoying and ugly. 

On the other other hand, we probably want to be consistent about having closing punctuation that wraps indented content live on its own line (eg; `>` in an opening JSX element, `)` for a function call – though I haven't looked at the behavior of that one recently). 

I don't have an opinion here myself, but thought I should highlight it as potentially worth discussing.

Comment (rattrayalex) [inline tests/jsx/__snapshots__/jsfmt.spec.js.snap]: Oops, looks like this just got merged anyway! Presumably the issue I highlighted above will get brought up as an issue later if people don't like it, and can be discussed there.

Comment (lydell) [inline tests/jsx/__snapshots__/jsfmt.spec.js.snap]: A random opinion: I've always used "the dangling `}`" style. ¯\\_(ツ)\_/¯

Comment (jlongster) [inline tests/jsx/__snapshots__/jsfmt.spec.js.snap]: @vjeux what do you think about the above? I haven't used JSX enough to have an opinion.

Comment (rattrayalex) [inline tests/jsx/__snapshots__/jsfmt.spec.js.snap]: Hah, hey @lydell ! Good to see you here from coffeescript6 discussions. 

Would love your thoughts on an alt-js lang I'm working on if you have some time to chat? rattray.alex@gmail.com 

(apologies all for the sidechat)

**Comment by vjeux** [comment]
Yeah, I'm unsure about this issue. I've personally seen both in the fb codebase and could go either way. This one was the easiest to implement (which is in itself not a valid reason)

**Pull request merged**